src/main/java/org/jabref/gui/journals/AbbreviateAction.java

```diff
@@ -53,10 +63,20 @@ public void run() {
 
                 return false;
             };
+            tasks.add(callable);
+        }
 
-            boolean result = JabRefExecutorService.INSTANCE.executeAndWait(callable);
-            if (result) {
-                count++;
+        // Execute the callables and wait for the results.
+        List<Future<Boolean>> futures = JabRefExecutorService.INSTANCE.executeAll(tasks);
+
+        // Evaluate the results of the callables.
+        for (Future<Boolean> future : futures) {
```

Comment:
According to https://stackoverflow.com/a/1250668/873282, this is not required. See also above.